attw.json

```diff
@@ -347,6 +347,7 @@
         "url-state",
         "use-combined-reducers",
         "use-global-hook",
+        "use-sync-external-store",
```

Comment:
I don't want to add types for the entrypoints that only exist for backwards compat (e.g. `/shim.js`) so that they don't pollute autocomplete.

Reply:
I really do not think we should be adding to this list. The intent is for this file to slowly be emptied as we fix things, not grow.

Reply:
I don’t see `./shim.js` as one of the exports: https://app.unpkg.com/use-sync-external-store@1.5.0/files/package.json

I don’t know anything about this package, but are you saying that some of the entrypoints should _never_ be imported by _anyone_? By only listing a subset of the entrypoints, you’re not just affecting autocomplete, you’re preventing users from writing those imports by hand. It’s not an implicit-any untyped module error; it’s a module-does-not-exist error.